www/index.php

```diff
@@ -237,68 +244,25 @@
 				</div>
 			</div>
 			
-			<div id="setRootNode" class="fieldgroup" style="">
+			<div id="setRootNode" class="fieldgroup" style="display: none;">
```

Comment:
In case of buttons it is usually fine because they signify an event and can be named on what would happen if someone clicks them.